GVRf/Framework/framework/src/main/backends/oculus/org/gearvrf/GVRViewManager.java

```diff
@@ -663,6 +663,7 @@ public void beforeDrawEyes() {
             if (mActivity.getAppSettings().showLoadingIcon) {
                 mSplashScreen = mScript.createSplashScreen();
                 if (mSplashScreen != null) {
+                    getMainScene().setUseMultiview(mActivity.getAppSettings().isMultiviewSet());
```

Comment:
There is a separate path where there is no splash screen requested and this flag is not going to be set properly in that case. And we shouldn't be using the scene for this flag since the scenes can be instantiated all the time. We don't want to keep track of all instances and try to set the flag for all.